README.md

```diff
@@ -80,6 +80,8 @@ These commands are grouped, so each group has a prefix but you can easily tab co
 
 - Custom openers need to be placed as a .txt file in the `openers` directory, in the same directory as `gpt3discord.py`
 
+- Can use .json files in the `{"text": your prompt, "temp":0, "top_p":0,"frequency_penalty":0,"presence_penalty":0}` format to include permanent overrides

```

Comment:
Could you fix conflicts and maybe add a bit more info on how to use this? maybe with an example or tell them to refer to the sample .json file?

Reply:
Added some to the readme